mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/43 - Office view

Closes #43 

Scene inputs (context menu, camera controls, etc) should only work when mouse is in Office View with mouse over non-UI area of screen.

To test:
- Play Introductory scene
- Make sure camera controls work (scroll wheel for zoom, right click and drag for rotate)
- Make sure context menu works (right click for open, right click again to close if on root menu)
- Left click on computer/user to bring up their context menu (this will be replaced with object selection for my next task)
- Make sure buttons at top of Office View move camera between users/computers as expected
- Make sure u on keyboard cycles through users
- Make sure c and d on keyboard cycles between computers/devices

Notes:
- Currently cycling between computers and devices cycles through full list of both since it still uses Component's GetNextComponent functionality
